spikeinterface/core/job_tools.py

```diff
@@ -227,12 +233,15 @@ class ChunkRecordingExecutor:
 
     def __init__(self, recording, func, init_func, init_args, verbose=False, progress_bar=False, handle_returns=False,
                  n_jobs=1, total_memory=None, chunk_size=None, chunk_memory=None, chunk_duration=None,
-                 job_name=''):
-
+                 mp_context="fork", job_name=''):
+        if mp_context is None:
```

Comment:
if mp_context is None I would leave None.
I guess internally this avoid a call with mp.get_context('fork')

Reply:
agreed